portfolio/src/main/webapp/script.js

```diff
@@ -135,4 +135,37 @@ function createMap() {
               stylers: [{color: '#17263c'}]
             }]
       });
+
+    addLandmark(
+      map, -17.824858, 31.053028, 'Harare, Zimbabwe',
+      'I was born and raised in Harare, Zimbabwe.')
```

Comment:
have you tested this code? i feel like you should need semicolons after each call to addLandmark

Reply:
Yes I have tested the code. The code works for both with and without semicolons, but I am sure it should have semicolons.